spark/common/tools/maven/scalastyle_config.xml

```diff
@@ -0,0 +1,194 @@
+<scalastyle commentFilter="enabled">
+ <name>Scalastyle standard configuration</name>
```

Comment:
Why do all these? We already have the same content here: https://github.com/apache/sedona/tree/master/tools/maven